core/templates/dev/head/pages/exploration_editor/exploration_editor.html

```diff
@@ -279,6 +279,10 @@
   <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_player/ExplorationPlayerStateService.js"></script>
   <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_player/PlayerConstants.js"></script>
   <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_player/AnswerClassificationService.js"></script>
+  <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_player/StateClassifierMappingService.js"></script>
+  <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_player/PredictionAlgorithmRegistryService.js"></script>
+  <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_player/PredictionAlgorithmSampleService.js"></script>
```

Comment:
@seanlip I have one idea. We can incorporate these (prediction service) dependencies similar to how interaction dependencies are added. We will add prediction service dependencies in `interaction.py` file. Then we can add a helper function in `interaction_registery` which will get us deduplicated prediction service dependencies. We can store all these dependencies in `extensions/dependencies/prediction_service_dependencies`, from where we will get their respective html files and directly include them in frontend. (The html files will contain bunch of `<script src=""></script>` tags.)
How does this scheme sound? If its okay then I will start working on it in a separate PR.